[Docs] coupled solid documentation (#1539)
GEOSX/GEOSX: src/coreComponents/constitutive/docs/KozenyCarmanPermeability.rst

.. _KozenyCarmanPermeability:

####################################################
Kozeny-Carman Permeability Model
####################################################

Overview
======================

> In the Kozeny-Carman equation, the permeability of a porous medium is governed by multiple key parameters, including porosity, grain size, and grain shape:

Reviewer: ```suggestion
In the Kozeny-Carman model, the permeability of a porous medium is governed by several key parameters, including porosity, grain size, and grain shape:
```

.. math::

>    k =  \frac{{{s_{\epsilon}}{D_p}}^2 {\phi}^3} {150 {1 - \phi}^2} 

Reviewer: Add reference

> where :math:`s_{\epsilon}` is the sphericity of the particles, :math:`D_p` is the particle diameter, :math:`\phi` is the porosity of the porous medium.

Reviewer 2: ```suggestion
where :math:`s_{\epsilon}` is the particle sphericity, :math:`D_p` the particle diameter and :math:`\phi` the porosity of the porous medium.
```

Parameters
======================

The Kozeny-Carman Permeability Model can be called in the
``<Constitutive>`` block of the input XML file.

> This permeability model must be assigned a unique name via

Reviewer: ```suggestion
This permeability model must be assigned a unique name via the
```
